rest/doc_api.go

```diff
@@ -24,26 +25,42 @@ import (
 func (h *handler) handleGetDoc() error {
 	docid := h.PathVar("docid")
 	revid := h.getQuery("rev")
-	includeRevs := h.getBoolQuery("revs")
 	openRevs := h.getQuery("open_revs")
 
-	// What attachment bodies should be included?
-	var attachmentsSince []string = nil
-	if h.getBoolQuery("attachments") {
-		atts := h.getQuery("atts_since")
-		if atts != "" {
-			err := json.Unmarshal([]byte(atts), &attachmentsSince)
-			if err != nil {
-				return base.HTTPErrorf(http.StatusBadRequest, "bad atts_since")
+	// Check whether the caller wants a revision history, or attachment bodies, or both:
+	var revsLimit = 0
+	var revsFrom, attachmentsSince []string
+	{
```

Comment:
I haven't seen this kind of use of braces to create a statement without an operator/function before.  Are you doing something with the scoping of the variables that requires it (I couldn't spot anything), or is it just for functional grouping?